src/archivematicaCommon/lib/archivistsToolkit/client.py

```diff
@@ -22,6 +22,23 @@ def __init__(self, host, user, passwd, db):
             logger.exception('Error connecting to ATK database')
             raise
 
+    def get_levels_of_description(self):
+        """
+        Returns an array of all levels of description defined in this Archivist's Toolkit instance.
+        """
+        if not hasattr(self, 'levels_of_description'):
+            cursor = self.db.cursor()
+            levels = set()
+            cursor.execute("SELECT distinct(resourceLevel) FROM Resources")
```

Comment:
Nope. The rest of the file uses raw SQL too; this was ported pretty straight from the old code.

Didn't seem appropriate to use Django models for this, since Django assumes a greater degree of ownership over the schema and this is an external app's DB.